R/standardise_titles.R

```diff
@@ -19,22 +19,24 @@
 #' @importFrom english ordinal words
 #' @importFrom stringr str_count str_squish str_to_title
 #' @importFrom utils as.roman
-#' @importFrom stringi stri_trans_general
 #' @import dplyr
```

Comment:
How come cld2 and translateR are not listed here? We're just making this a conditional suggests import?

Reply:
The function uses `depends()` to import and/or load these packages only if an API key is provided. Would you like me to add these to the function documentation?

Reply:
Ah, ok then.